ios/Classes/CameraPreview/CameraPreview.m

```diff
@@ -113,7 +114,19 @@ - (void)setCameraPresset:(CGSize)currentPreviewSize {
     }
     [_captureSession setSessionPreset:presetSelected];
     _currentPresset = presetSelected;
-    
+}
+
+/// Set camera video size
+- (void)setCameraVideoPresset:(CGSize)currentPreviewSize {
```

Comment:
This should be named `setCameraPreviewPresset`, and the other one should have this name I think. Also `currentPreviewSize` is misleading, because it is the new/requested size.

```suggestion
- (void)setCameraPreviewPreset:(CGSize)requestedSize {
```

Reply:
I updated the code to fix naming confusion and moved the assignment of _currentPreviewSize to the other method. I think this should be alright now, can you re-review?